test/models/hold_test.rb

```diff
@@ -70,6 +70,33 @@ class HoldTest < ActiveSupport::TestCase
     assert(hold.invalid?)
   end
 
+  test 'active_or_expired scope shows expected records' do
+    assert_equal ["active", "expired"], Hold.active_or_expired.map(&:status).uniq.sort
+    hold = holds(:valid)
+    hold.status = :active
+    hold.save
+    assert Hold.active_or_expired.pluck(:id).include?(hold.id)
+    hold.status = :expired
+    hold.save
+    assert Hold.active_or_expired.pluck(:id).include?(hold.id)
+    hold.status = :released
+    hold.save
+    assert_not Hold.active_or_expired.pluck(:id).include?(hold.id)
+  end
+
+  test 'ends_today_or_before scope shows expected records' do
```

Comment:
Similar to above not on tests likely being easier to follow if split up. Alternately adding comments to the tests to show what each assertion is checking would work as well.